tests/store/artifact/test_gcs_artifact_repo.py

```diff
@@ -99,6 +100,9 @@ def test_list_artifacts_with_subdir(gcs_mock):
     gcs_mock.Client.return_value.bucket.return_value.list_blobs.return_value = mock_results
 
     artifacts = repo.list_artifacts(path=dir_name)
+    gcs_mock.Client().bucket().list_blobs.assert_called_with(
+        prefix=posixpath.join(artifact_root_path[1:], dir_name, ""), delimiter="/"
```

Comment:
Better readability since we are checking that it ends with a "/"
```suggestion
        prefix=posixpath.join(artifact_root_path[1:], "model/"), delimiter="/"
```

Reply:
Can we also avoid the `[1:]` in artifact_root_path by modifying it at the start of the test?